content/en/database_monitoring/connect_dbm_and_apm.md

```diff
@@ -415,3 +415,4 @@ View historical performance of similar queries to those executed in your trace,
 [22]: https://docs.oracle.com/javase/8/docs/technotes/guides/jdbc/
 [23]: https://github.com/DataDog/dd-trace-java
 [24]: https://learn.microsoft.com/en-us/dotnet/framework/data/adonet/ado-net-overview
+[25]: https://learn.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlcommand.commandtype?view=dotnet-plat-ext-7.0#remarks:~:text=[…]%20should%20set
```

Comment:
This link leads me to `https://learn.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlcommand.commandtype?view=dotnet-plat-ext-7.0#remarks`
Is this the intended link?

Reply:
Yes, I wanted to give any customer an idea of the type that doesn't get supported.

Reply:
@estherk15 is there a way to add this as a footnote marker on each item in the `Library or framework` column with the note at the bottom of the table?

Reply:
@estherk15 this change specifically would apply to .NET.